docs/cudf/source/internals.md

```diff
@@ -27,7 +27,9 @@ A column is composed of the following:
   to 64 bytes.
 * A tuple of **children** columns, which enable the representation complex types such as
   columns with non-fixed width elements (e.g., strings, lists).
-* If the Column represents a slice, an integer **offset** into the first element of the slice
+* A *size* indicating the number of elements in the Column  
+* If the Column represents a slice, an integer **offset** into the first element of the slice.
```

Comment:
This makes it seem liked only sliced columns have an offset, whereas there's always an `offset`, which is just `0` in most cases.

Reply:
Fixed